### DISCUSSION TITLE
Realtime charts without rerendering with smooth animation

### DISCUSSION THREAD

**Discussion by Waishnav** [apexcharts/apexcharts.js#4213] [General]
Hi!
I'am trying to figure out how to connect apexchart with async data without full re-rendering.
Every few seconds after, I receive new point for area chart, after new data comes it is triggering re-rendering :(

But my questions are:

1. How to apply only new data without full re-rendering?
2. How to move xAxis smoothly? [d3 example](https://bost.ocks.org/mike/path/)

I found only one library (highcharts) having realtime chart support without rerendering the all data points.
https://jsfiddle.net/sdoomz/ams3jyj2/

**Comment by technyon** [comment]
Late answer, but you can call appendSeries() to add your new data.
